### PR TITLE
Provide details on how to manually test staging addons

### PR DESCRIPTION
**Pull request by nfnt**
The existing doc is a bit vague on this

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -17,6 +17,44 @@ Feature branches should never interact directly with `master` or `staging` branc
 
 Pull Requests of feature branches should be opened against the `dev` branch. 
 
+## Deploying `staging` addons with Kommander
+
+This is best done on the AWK Soak cluster, its Kommander already has a `kudo-staging` project.
```
Comment (armandgrillet) [inline RELEASE.md:22]:
```suggestion
This is best done on the soak cluster running on AWS, its Kommander already has a `kudo-staging` project.
```

File: RELEASE.md [hunk 2]
```diff
@@ -17,6 +17,44 @@ Feature branches should never interact directly with `master` or `staging` branc
...
+    url: https://github.com/mesosphere/kubeaddons-enterprise
+  status:
+    ready: true
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+```
+
+Once that's done, create a cluster on AWS using the konvoy CLI. Get that cluster's kubeconfig, attach the cluster in soak's Kommander, then add the cluster to the `kudo-staging` project.
```
Comment (armandgrillet) [inline RELEASE.md:54]:
```suggestion
Once that's done, create a cluster on AWS using the konvoy CLI. Get that cluster's kubeconfig, attach the cluster to soak's Kommander, then add the cluster to the `kudo-staging` project.
```

File: RELEASE.md [hunk 3]
```diff
@@ -17,6 +17,44 @@ Feature branches should never interact directly with `master` or `staging` branc
...
+
+You should now be able to view the staging addons in the projects catalog and deploy them on the attached cluster. Verify that the updated addons deploy successfully by adding them in Kommander and checking with `kubectl` that the respective instance resources are created on the attached cluster.
```
Comment (armandgrillet) [inline RELEASE.md:56]:
```suggestion
You should now be able to view the staging addons in the project's catalog and deploy them on the attached cluster. Verify that the updated addons deploy successfully by adding them in Kommander and checking with `kubectl` that the respective instance resources are created on the attached cluster.
```

**Review by armandgrillet — APPROVED** [review]
LGTM, just a few nits.

**Review by armandgrillet — APPROVED** [review]

File: RELEASE.md [hunk 4]
```diff
@@ -17,6 +17,44 @@ Feature branches should never interact directly with `master` or `staging` branc
 
 Pull Requests of feature branches should be opened against the `dev` branch. 
 
+## Deploying `staging` addons with Kommander
+
+This is best done on the soak cluster running on AWS, its Kommander already has a `kudo-staging` project.
+First, we need to update the `AddonRepository` of the `kudo-staging` project to point to the stating branch. For that, generate a cluster token in soak Konvoy UI to be able to use `kubectl`. There should be a `kudo-stating-%something%` namespace containing the `kubeaddons-enterprise` addonrepository. Edit this and change its `spec.ref` to the staging branch, e.g.:
```
Comment (meichstedt) [inline RELEASE.md:23]:
```suggestion
First, we need to update the `AddonRepository` of the `kudo-staging` project to point to the staging branch. For that, generate a cluster token in soak Konvoy UI to be able to use `kubectl`. There should be a `kudo-staging-%something%` namespace containing the `kubeaddons-enterprise` addonrepository. Edit this and change its `spec.ref` to the staging branch, e.g.:
```

**Review by meichstedt — COMMENTED** [review]
Some typos

**Review by meichstedt — APPROVED** [review]
lgtm

**Review by DominikDary — APPROVED** [review]

**Review by armandgrillet — APPROVED** [review]

**Pull request merged**